aid/styles/Event.scss

```diff
@@ -60,12 +61,23 @@
 @media #{$mobile} {
     .Event {
         @include flexbox(column, nowrap, flex-start, flex-start);
+        padding-left: 5px;
```

Comment:
? i don't actually remember why this is here LOL 

TODO: figure this out, prob remove it in next commit... zz